### ISSUE TITLE
TSC linter fails when no errors

### ISSUE THREAD

**Issue by coatesap** [wearerequired/lint-action#732]
Currently ESLint and Prettier linters seem to be working correctly in our workflow, but Typescript / tsc is failing even when there aren't any errors.

Our workflow step: 

```yaml
    - name: Run linters
      uses: wearerequired/lint-action@v2
      with:
        eslint: true
        prettier: true
        tsc: true
        eslint_dir: src/
        prettier_dir: src/
        tsc_dir: src/
```

The raw log looks like this:

```
2023-10-12T08:50:21.7972887Z ##[group]Run TypeScript
2023-10-12T08:50:21.7974380Z Verifying setup for TypeScript…
2023-10-12T08:50:22.3702825Z Verified TypeScript setup
2023-10-12T08:50:22.3703299Z Will use TypeScript to check the files with extensions ts
2023-10-12T08:50:22.3704341Z Linting files in /root/actions-runner/_work/{REPO_NAME}/{REPO_NAME}/src/ with TypeScript …
2023-10-12T08:50:25.9688569Z TypeScript found no issues (failure)
2023-10-12T08:50:25.9689272Z ##[endgroup]
```

**Comment by coatesap** [comment]
Issue is still present.
